src/main/java/io/github/oliviercailloux/j_voting/preferences/interfaces/Preference.java

```diff
@@ -0,0 +1,42 @@
+package io.github.oliviercailloux.j_voting.preferences.interfaces;
+
+import java.util.Set;
+
+import com.google.common.graph.Graph;
+
+import io.github.oliviercailloux.j_voting.Alternative;
+import io.github.oliviercailloux.j_voting.Voter;
+
+/**
+ *
+ * Preference Interface is used to define the order chosen by a Voter between
+ * some alternatives.
+ *
```

Comment:
Préciser : a preference is empty iff it contains no alternative, equivalently, iff it contains no edge.